backend/faq/migrations/0004_alter_faqentry_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.0.8 on 2022-10-20 14:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Why do we have this change here? Was this migration simply forgotten in a previous change?

Reply:
Exactly